src/Cache.php

```diff
@@ -30,10 +30,10 @@ class Cache
      * on success.
      *
      * @param  string $filename
-     * @param  \Laminas\Server\Server $server
+     * @param  \Laminas\Server\ServerInterface $server
      * @return bool
      */
-    public static function save($filename, Server $server)
+    public static function save($filename, ServerInterface $server)
```

Comment:
BC Break - we cannot change signature. We can do it in v3, not here, unfortunately.

Reply:
Reverted this change.